src/App.tsx

```diff
@@ -27,69 +29,161 @@ type ReorderOptions = {
 };
 
 // Use this function in the render method to prepare goods
+let visibleGoods = [...goodsFromServer];
+
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  if (isReversed === true) {
+    return visibleGoods.reverse().map((good) => {
+      return (
+        <li data-cy="Good" key={good}>
+          {good}
+        </li>
+      );
+    });
+  }
+
+  switch (sortType) {
```

Comment:
Use the SortType enum values instead of numbers for better code readability.

Reply:
![image](https://github.com/mate-academy/react_list-of-goods/assets/80322190/511dbd96-9842-44d2-9287-7c1542a5c98b)
![image](https://github.com/mate-academy/react_list-of-goods/assets/80322190/063302f8-5db0-49f4-bf7b-4a63aa49bd47)
please give me advice, how to use enum in my switch case

Reply:
thanks, i already got it